### PR TITLE
WFLY-13557 Clustering TS: Support for test execution parallelization …

### PR DESCRIPTION
**Pull request by rhusar**
Resolves
https://issues.redhat.com/browse/WFLY-13557
https://issues.redhat.com/browse/WFLY-13563
https://issues.redhat.com/browse/WFLY-13564

### CODE REVIEW

File: docs/src/main/asciidoc/_testsuite/WildFly_Testsuite_Overview.adoc [hunk 1]
```diff
@@ -119,16 +120,79 @@ Divided into three Surefire executions:
...
+could be used to split the clustering tests into two executions of similar execution time, the first node can run the first half of the tests in sub-packages, e.g.:
 
-Sets up a cluster of two nodes.
+    $ ./integration-tests.sh clean install -Dts.noSmoke -Dts.clustering -P \!ts.clustering.cluster.fullha.profile,\!ts.clustering.single.profile,\!ts.clustering.byteman.profile -Dts.surefire.clustering.ha.additionalExcludes=affinity\|cdi\|dispatcher\|ejb\|ejb2\|group\|jms\|jpa
+
+while another node can concurrently run all the other profiles and the other half of sub-packages:
+
+    $ ./integration-tests.sh clean install -Dts.noSmoke -Dts.clustering -Dts.surefire.clustering.ha.additionalExcludes=jsf\|provider\|registry\|singleton\|sso\|web\|xsite
+
+If the test packages get out of sync with the excludes this will result in a test running multiple times, rather than tests being omitted.
```
Comment (rhusar) [inline docs/src/main/asciidoc/_testsuite/WildFly_Testsuite_Overview.adoc:170]: @kabir This is the section that would interest you.

**Comment by rhusar** [comment]
@pferraro please review if this work for you.

**Comment by kabir** [comment]
For the record this works great on my cross component CI stuff - thanks @rhusar

**Review by pferraro — APPROVED** [review]

**Comment by rhusar** [comment]
Just to note, this *shaves off* about 15-20 minutes off execution of the clustering tests (on our CI even more than that). This is due to the fact we no longer run all tests on full-ha, but only the relevant ones.

**Pull request merged**